packages/experimental/Menu/package.json

```diff
@@ -27,7 +27,8 @@
   },
   "dependencies": {
     "@fluentui-react-native/adapters": ">=0.8.5 <1.0.0",
-    "@fluentui-react-native/framework": "0.7.23",
+"@fluentui-react-native/framework": "0.7.23",
+    "@fluentui-react-native/interactive-hooks": ">=0.15.3 <1.0.0",
```

Comment:
spacing is off?

Reply:
Uh yeah idk how prettier didn't catch that